e2e/exhaustive_suite_test.go

```diff
@@ -0,0 +1,80 @@
+// SPDX-FileCopyrightText: The RamenDR authors
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e_test
+
+import (
+	"testing"
+
+	"github.com/ramendr/ramen/e2e/deployers"
+	"github.com/ramendr/ramen/e2e/testcontext"
+	"github.com/ramendr/ramen/e2e/workloads"
+)
+
+// Deployers = {"Subscription", "AppSet", "Imperative"}
+// Workloads = {"Deployment", "STS", "DaemonSet"}
+// Classes   = {"rbd", "cephfs"}
+
+func Exhaustive(t *testing.T) {
```

Comment:
since current test output already contains t.Name, so I think it is not necessary to print it again. please see the output sample.

```
=== CONT  TestSuites/Exhaustive/Deployment/Subscription
2024-04-25T23:23:57.353+0800	INFO	deployers/applicationset.go:17	enter Deploy Deployment/Appset
=== RUN   TestSuites/Exhaustive/Deployment#01/Appset/Enable
2024-04-25T23:23:57.353+0800	INFO	dractions/actions.go:21	enter EnableProtection Deployment/Appset
=== RUN   TestSuites/Exhaustive/Deployment/Subscription/Deploy
=== RUN   TestSuites/Exhaustive/Deployment#01/Appset/Failover
2024-04-25T23:23:57.353+0800	INFO	deployers/subscription.go:36	enter Deploy Deployment/Subscription
=== RUN   TestSuites/Exhaustive/Deployment/Subscription/Enable
2024-04-25T23:23:57.353+0800	INFO	dractions/actions.go:21	enter EnableProtection Deployment/Subscription
=== RUN   TestSuites/Exhaustive/Deployment/Subscription/Failover
2024-04-25T23:23:57.353+0800	INFO	dractions/actions.go:33	enter Failover Deployment/Subscription
=== RUN   TestSuites/Exhaustive/Deployment/Subscription/Relocate
2024-04-25T23:23:57.353+0800	INFO	dractions/actions.go:39	enter Relocate Deployment/Subscription
```

Reply:
Ack! noted.